packages/venia-ui/lib/components/WishlistPage/__tests__/wishlistEditFavoritesListDialog.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { createTestInstance } from '@magento/peregrine';
+
+import WishlistEditFavoritesListDialog from '../wishlistEditFavoritesListDialog';
+
+jest.mock('../../../classify');
+jest.mock('../../Dialog', () => 'Dialog');
+jest.mock('../../FormError', () => 'FormError');
+
+const props = {
+    formErrors: new Map([]),
+    isOpen: false,
+    isEditInProgress: false,
+    onCancel: jest.fn(),
```

Comment:
We can squeeze a little extra value out of snapshots by naming these mocks.

```suggestion
    onCancel: jest.fn().mockName('onCancel'),
```